src/builder/javascript.js

```diff
@@ -1,16 +1,29 @@
 import { transformFile } from 'babel-core'
 import { default as deferred } from 'thenify'
 import { default as compat } from 'babel-plugin-add-module-exports'
-import { default as es2015 } from 'babel-preset-es2015'
-import { default as umd } from 'babel-plugin-transform-es2015-modules-umd'
+import { default as ecmascript } from 'babel-preset-latest'
+
+const babelrc = process.env.NODE_ENV !== 'test'
 
 export default function configure(pkg, opts) {
   return async function process(name, file) {
-    let presets = [es2015]
-      , plugins = [umd]
+    const { es2017
+          , modules
+          , ['add-module-exports']: addModuleExports
+          } = opts.flags
+
+    let presets = [
```

Comment:
nah, you are right - this PR is :+1: and we can then look at how to best solve it and maybe a few more comments / approaches.